plugins/com.google.cloud.tools.eclipse.appengine.deploy.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,15 @@
+Manifest-Version: 1.0
+Bundle-ClassPath: .
+Bundle-ManifestVersion: 2
+Bundle-Name: Test
```

Comment:
Should we give a more descriptive names to test bundles? (This would mean updating the other test bundle names as well).